packages/mobile/src/screens/user-list-screen/UserListScreen.tsx

```diff
@@ -31,7 +31,8 @@ export const UserListScreen = (props: UserListScreenProps) => {
     <Screen
       variant='white'
       title={isTippingEnabled ? undefined : title}
-      headerTitle={isTippingEnabled ? headerTitle : undefined}>
+      headerTitle={isTippingEnabled ? headerTitle : undefined}
+      topbarRight={null}>
```

Comment:
just curious - where did topbarRight default to the search icon in the first place? can't seem to find it..

Reply:
yeah that's a good point, it's super hard to find where this is first off, and secondly it prob doesn't make sense to have search as a default thing anyway...

Reply:
its here tho: https://github.com/AudiusProject/audius-client/blob/main/packages/mobile/src/screens/app-screen/useAppScreenOptions.tsx